src/config/app.ts

```diff
@@ -13,7 +13,6 @@ export const ACTION = 'laminas/laminas-continuous-integration-action@v1';
 export enum ComposerDependencySet {
     LOWEST = 'lowest',
     LOCKED = 'locked',
-    LATEST = 'latest',
```

Comment:
I would not remove this. It should be possible to generate Jobs with `latest`, but that wont be the default tho besides opt-in. 

I still see this being necessary as something has to check latest, just not every PR.